packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -71,6 +71,18 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
     objectNameSingular: relationObjectNameSingular,
   });
 
+  const onCreateWithInput = isDefined(onCreate)
```

Comment:
handleCreate = () => {
  if(!isDefined(onCreate)) {
    return
  } 

  if (onCreate.length > 0) {
          (onCreate as (searchInput?: string) => void)(
            relationPickerSearchFilter,
          );
        } else {
          (onCreate as () => void)();
        }
}

Reply:
That would make it look like handleCreate is defined (with () => {return}). But I'll try to simplify!